<?xml version="1.0"?>

<launch>
  <node name="person_blob_ros" pkg="person_blob_ros" type="find_blob_srv.py" output="screen">
    <rosparam command="load" file="$(find person_blob_ros)/launch/check.yaml"/>
  </node>
</launch>
